bundler/lib/bundler/cli.rb

```diff
@@ -381,6 +381,8 @@ def info(gem_name)
       "Make binstubs that can work without the Bundler runtime"
     method_option "all", :type => :boolean, :banner =>
```

Comment:
Perhaps this should be renamed to `all-gems` (and have `--all` added as an alias)?

Reply:
Not sure, I think `--all` is mostly fine?